darwin/item.py

```diff
@@ -92,12 +92,11 @@ def parse(cls, raw: Dict[str, Any]) -> "DatasetItem":
                 "archived": raw["archived"],
                 "filesize": sum(file.get("size_bytes", 0) for file in raw["slots"]),
                 "dataset_id": raw["dataset_id"],
-                "dataset_slug": "n/a",
+                "dataset_slug": dataset_slug,
                 "seq": None,
-                "current_workflow_id": None,
-                "current_workflow": None,
+                "current_workflow_id": raw.get("workflow_data", {}).get("workflow_id"),
+                "current_workflow": raw.get("workflow_status"),
```

Comment:
`workflow_status` is not a workflow data really. I think this should simply be `raw.get("workflow_data")`.